indico/modules/events/editing/models/revisions.py

```diff
@@ -165,3 +165,10 @@ def __repr__(self):
     @locator_property
     def locator(self):
         return dict(self.editable.locator, revision_id=self.id)
+
+    def get_spotlight_file(self):
+        files = [file for file in self.files if file.file_type.publishable]
+        return files[0] if len(files) == 1 else None
```

Comment:
```suggestion
        return files[0] if len(files) else None
```

You can leave this as is, up to your criteria ;) but here's a suggestion I saw the other day for a simple and pretty cool idiom to return the first element:
```suggestion
        return next(iter(files), None)
```

Reply:
@plourenco yeah, that's quite nice looking, but it has two drawbacks IMHO:
 * it creates a whole iterator for what is a single operation;
 * it makes the whole thing less readable

Reply:
`if len(files)` doesn't look like a good idea to me in any case - you'd use `if files` there. But I think this check here is especially about checking that there is exactly one file, not that there are any number of files.

Reply:
Yes, we want to check if there's exactly one file, which is this case is our spotlight file.